grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', function () {
+    // grunt addition for htmllint
+  });
+
+  grunt.registerMultiTask('l10n-json-to-html', 'Convert l10n JSON files to HTML', function () {
+    var done = this.async();
+    this.files.forEach(function (file) {
+      var content = '';
+      var src = file.src[0];
+      var path = src.split('/');
+      var locale = grunt.file.readJSON(src);
+      for (var val in locale) {
+        var value = locale[val];
+        if (typeof value !== 'object' && value !== '' && isNaN(value)) {
```

Comment:
there are a bunch of files that have only the years in them, no use writing them out, like 1996,1997,1998 etc